edu.cuny.hunter.streamrefactoring.core/src/edu/cuny/hunter/streamrefactoring/core/analysis/StreamStateMachine.java

```diff
@@ -1000,6 +1005,9 @@ private void propagateStreamInstanceProperty(Collection<InstanceKey> streamInsta
 		// for each rule.
 		SubMonitor ruleMonitor = subMonitor.split(70, SubMonitor.SUPPRESS_NONE).setWorkRemaining(ruleArray.length);
 
+		// slicing
+		Graph<CGNode> prunedCallGraph = pruneCallGraph(engine.getCallGraph());
```

Comment:
Can we have a log on the size that we are saving? LIke number of nodes and edges before and after the refactoring?

Reply:
Yes.